checkov/cloudformation/checks/graph_checks/aws/SagemakerNotebookEncryption.yaml

```diff
@@ -0,0 +1,10 @@
+metadata:
+  id: "CKV2_AWS_24"
+  name: "Ensure all data stored in the Sagemaker Notebook is securely encrypted at rest"
```

Comment:
the name is misleading here, the notebook instances are encrypted by default. a text like
```
"Ensure SageMaker Notebook is encrypted at rest using KMS CMK"
```
would be nicer :)

Reply:
@gruebel I agree with you, I do want to keep it uniform with the matching terraform check - https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/SagemakerNotebookEncryption.py
Do you think I should prefer clarity over consistency?

Reply:
I renamed both checks :)